src/SFML/Graphics/Font.cpp

```diff
@@ -583,11 +583,14 @@ Glyph Font::loadGlyph(Uint32 codePoint, unsigned int characterSize, bool bold, f
         // pollute them with pixels from neighbors
         const unsigned int padding = 1;
 
+        width += 2 * padding;
+        height += 2 * padding;
```

Comment:
(sorry I was away the whole week, I try to catch up)

What's the reason for adding 2 * padding to width and height here, and then constantly subtracting it everywhere width and height are used?